main/ertp/ERTP-Guide/PursesAndPayments.md

```diff
@@ -0,0 +1,301 @@
+  - Returns a `brand`, indicating what kind of digital asset the
+  `payment` purports to be. Since a `payment` is not trusted, this
+  result should be treated with suspicion and verified before 
+  use. If any operation by the `issuer` for that `brand` done on the `payment` succeeds, it verifies the `payment`.
+  For example, `E(issuer).isLive(payment)`.
+
+Other objects' `payment`-related methods:
+
+- [`issuer.getAmountOf(payment)`](https://agoric.com/documentation/ertp/api/issuer.html#issuer-getamountof-payment)
+  - Get a description of a `payment` balance as an `amount`. The `payment` itself is not trusted,
+    so you must use the `issuer` method associated with its `brand` to be sure of getting the
+    true value. 
+  ` ```js
+    const { quatloosIssuer, quatloosMint, quatloosAmountMath } = makeIssuerKit('quatloos');
+    const quatloosPayment = quatloosMint.mintPayment(quatloosAmountMath.make(100));
+    quatloosIssuer.getAmountOf(quatloosayment); // returns an amount with 100 value `quatloos` brand
+    ```
+- [`issuer.burn(payment, optAmount)`](https://agoric.com/documentation/ertp/api/issuer.html#issuer-burn-payment-optamount)
+  - Burn all of the digital assets in the `payment`. `optAmount` is
+    optional but if present, the `payment` balance must be equal to
+    it. If `payment` is a promise, the operation proceeds after it resolves. 
+  - ```js
+    const { quatloosIssuer, quatloosMint, quatloosAmountMath } = makeIssuerKit('quatloos');
+    const amountToBurn = quatloosAmountMath.make(10);
+    const paymentToBurn = quatloosMint.mintPayment(amountToBurn);
+    // burntAmount should equal 10
+    const burntAmount = quatloosIssuer.burn(paymentToBurn, amountToBurn);
+    ```
+- [`issuer.claim(payment, optAmount)`](https://agoric.com/documentation/ertp/api/issuer.html#issuer-claim-payment-optamount)
+  - Transfer all assets from the `payment` to a returned new `payment`
+    and burn the original. No other references to the argument `payment` survive, so 
+    the new `payment` is the exclusive one. If `optAmount` is
+    present, the `payment` balance must be equal to it or it throws
+    an error. If `payment` is a promise, the operation proceeds after it resolves. 
+  - ```js
+    const { quatloosMint, quatloosIssuer, quatloosAmountMath } = makeIssuerKit('quatloos');
+    const amountExpectedToTransfer = quatloosAmountMath.make(2);
+    const originalPayment = quatloosMint.mintPayment(amountExpectedToTransfer);
+    const newPayment = quatloosIssuer.claim(originalPayment, amountToTransfer);
+    ```
+- [`issuer.combine(paymentsArray)`](https://agoric.com/documentation/ertp/api/issuer.html#issuer-combine-paymentsarray)
+  - Combine multiple `payment`s into one, returned, `payment`. If any `payment` in
+  the array is a promise, the operation proceeds after every `payment`
+  resolves. All `payments` in the array are burned.
+  - ```js
```

Comment:
? I don't see the problem here. Maybe move it to before the preceding "promise" line? I added "upon successful completion" at the end, does that address it? Because from what I can tell, the original payments in that array of payments do all get burned in favor of the combined payment result.